webapp/lib/charts.dart

```diff
@@ -130,25 +133,29 @@ class DailyTimeseriesLineChartView {
             ..distribution = 'linear'
             ..bounds = 'ticks'
             ..time = (new chartjs.TimeScale(unit: 'day'))
-        ],
-        yAxes: [
-          new chartjs.ChartYAxe()
-            ..ticks = (new chartjs.LinearTickOptions()..beginAtZero = true)
+      ],
+      yAxes: [
+        new chartjs.ChartYAxe()
+          ..ticks = (new chartjs.LinearTickOptions()..beginAtZero = true)
         ]),
       hover: new chartjs.ChartHoverOptions()..animationDuration = 0
     );
 
-    var chartConfig = new chartjs.ChartConfiguration(type: 'line', data: chartData, options: chartOptions);
+    var chartConfig = new chartjs.ChartConfiguration(
+        type: 'line', data: chartData, options: chartOptions);
     chart = chartjs.Chart(canvas.getContext('2d'), chartConfig);
   }
 
-  void updateChart(Map<DateTime, int> updatedCountsPerDay) {
+  void updateChart(List<Map<DateTime, int>> updatedCountsPerDayList) {
     List<chartjs.ChartPoint> timeseriesPoints = [];
-    List<DateTime> sortedDateTimes = updatedCountsPerDay.keys.toList()
-      ..sort((t1, t2) => t1.compareTo(t2));
-    for (var datetime in sortedDateTimes) {
-      var value = updatedCountsPerDay[datetime];
-      timeseriesPoints.add(new chartjs.ChartPoint(t: datetime.toIso8601String(), y: value));
+    for (var updatedCountsPerDay in updatedCountsPerDayList) {
```

Comment:
I'm not sure this does what it's supposed to... I think `timeseriesPoints` should be defined inside the for loop, ie. for each dataset you should have a new set of timeseries points. And then `chartData.datasets[index].data ...` from below should also be inside the loop. Otherwise what this method seems to do, is to just accumulate all data points into the first dataset

Reply:
I think the reasoning there was to enable us to chart two lines at the same time unless I got that wrong. Does that make sense?

Reply:
![Screenshot 2020-03-30 at 10 44 46](https://user-images.githubusercontent.com/3750869/77887910-13ab7d00-7274-11ea-8d21-280587936f4a.png)

Reply:
That's from having the following code
`case UIAction.systemEventsDataUpdated:
      Map<DateTime, int> data = new Map.fromIterable(systemEventsDataList,
          key: (item) => (item as model.SystemEventsData).timestamp,
          value: (item) => 1);
          Map<DateTime, int> data2 = new Map.fromIterable(systemEventsDataList,
          key: (item) => (item as model.SystemEventsData).timestamp,
          value: (item) => 2);
      view.contentView.restartSystemEventTimeseries.updateChart([data, data2]);
      break;`

Reply:
But I've put them inside the loop

Reply:
As discussed offline, your last commit fixes this, thanks!